dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -28,6 +31,8 @@ type NonInlineProps = {
 	index?: never;
 	shouldHideReaderRevenue?: boolean;
 	isPaidContent?: boolean;
+	pillar?: ArticleTheme;
```

Comment:
Can we make this `format` and then access it via `format.theme`. We take the approach of passing `format` by convention even if the code only accesses one property from the object. Sorry, I wasn't clear about that before!